pkgs/tools/networking/keepalived/default.nix

```diff
@@ -1,34 +1,20 @@
-{ stdenv, fetchurl, openssl, net_snmp, libnl }:
+{ stdenv, fetchurl, pkgconfig, openssl }:
 
 stdenv.mkDerivation rec {
-  name = "keepalived-1.2.19";
+  name = "keepalived-${version}";
+  version = "1.3.2";
 
   src = fetchurl {
-    url = "http://keepalived.org/software/${name}.tar.gz";
-    sha256 = "0lrq963pxhgh74qmxjyy5hvxdfpm4r50v4vsrp559n0w5irsxyrj";
+    url = "https://github.com/acassen/keepalived/archive/v${version}.tar.gz";
+    sha256 = "0rsk21mwy07aas43xkmjbnxi8dknxvh8rcp3kb1s92w49jzxk40p";
   };
 
-  buildInputs = [ openssl net_snmp libnl ];
+  buildInputs = [ openssl ];
```

Comment:
Does passing pkgconfig here, allow removal of the preConfigure export of PKG_CONFIG?